packs/elastic/pack.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name : elastic
```

Comment:
k. I guess I need to rename it then.

On Wed, May 20, 2015 at 6:13 PM, Tomaz Muraus 
<notifications@github.com> wrote:
> In packs/elastic/pack.yaml:
> 
> > @@ -0,0 +1,10 @@
> > +---
> > +name : elastic
> I'd rather prefer to have a pack per product / tool, since in this 
> case, those products don't have much in common besides the parent 
> organization.
> 
> I don't think we could re-use any code and this means we would also 
> need to prefix actions with the service name - e.g. 
> elasticsearch.indices.optimize.
> 
> —
> Reply to this email directly or view it on GitHub.
>